manifests/2.7.0/opensearch-2.7.0-test-segment-replication.yml

```diff
@@ -0,0 +1,174 @@
+---
+schema-version: '1.0'
+name: OpenSearch
+ci:
+  image:
+    name: opensearchstaging/ci-runner:ci-runner-centos7-opensearch-build-v2
+    args: -e JAVA_HOME=/opt/java/openjdk-17
+components:
+  - name: index-management
+    integ-test:
+      build-dependencies:
+        - job-scheduler
+      test-configs:
+        - with-security
+        - without-security
+      additional-cluster-configs:
```

Comment:
Hey @gaiksaya I suspect the syntax as follows based on above setting like `build-dependencies`. Can you please check?
```
      additional-cluster-configs:
       - path.repo: [/tmp]
       - opensearch.experimental.feature.replication_type.enabled: true
       - cluster.indices.replication.strategy: 'SEGMENT'
  ```

Reply:
That's a list and not key-value pair. Looks like current schema wants `"additional-cluster-configs": {"type": "dict"},` that’s why its failing:
https://github.com/opensearch-project/opensearch-build/actions/runs/4514508042/jobs/7950692280?pr=3346

Reply:
Problem was with manifest naming. https://github.com/opensearch-project/opensearch-build/blob/main/src/ci_workflow/ci_manifests.py#L21-L24
Fixed the name it checked against the right schema now. 
Thanks!